shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -61,8 +63,13 @@ public Options getOptions() {
     return new Options().addOption(RECURSIVE_OPTION);
   }
 
+  private static final Pattern USER_PATTERN = Pattern.compile("(?<user>[\\w][\\w-]*)");
```

Comment:
I did some search and it seems the dollar sign `$` is legal to trail the user/group name.
It would be good to put in a comment that which standard we are following to construct this `USER_PATTERN`